CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ This project uses `makefile` for managing various chores, like dependency instal
 
 This project is a monorepo managed by [Lerna][lerna-home]. This means that each folder inside the [packages/][packages] directory represents a package published to npm. The root directory also contains _package.json_ file but this is only used for local development purposes and does not represent something that is ever published to npm (you might notice that it only contains _devDependencies_ and no prod deps 🤷‍♂️).
 
-## Commit messages
+## Commit Conventions
```

Comment:
Change it in the content above too, please.

Reply:
Fixed.